tests/tests_suite_test.go

```diff
@@ -35,26 +37,59 @@ var _ = AfterSuite(func() {
 	removeTestNamespaces()
 })
 
+func getPodLogs(podName, containerName string) (string, error) {
+	req := testClient.VirtClient.CoreV1().Pods(managerNamespace).GetLogs(podName, &corev1.PodLogOptions{
+		Container: containerName,
+	})
+	podLogs, err := req.Stream(context.TODO())
+	if err != nil {
+		return "", err
+	}
+	defer func(podLogs io.ReadCloser) {
+		err := podLogs.Close()
+		if err != nil {
+			panic(err)
+		}
+	}(podLogs)
+
+	buf := new(bytes.Buffer)
+	_, err = io.Copy(buf, podLogs)
```

Comment:
why to copy it first to buf and just directly use `podLogs.String()` ?

Reply:
podLogs is an io stream interface. This means it provides a way to read data sequentially, not as a complete in-memory string. Copying it to the buffer is a way to safely move the data to a string.